src/TableReader.jl

```diff
@@ -661,34 +673,34 @@ function _precompile_()
-    precompile(Tuple{typeof(TableReader.parse_datetime), Array{Union{Base.Missing, String}, 1}})
-    precompile(Tuple{typeof(TableReader.fillcolumn!), Array{Float64, 1}, Int64, TranscodingStreams.Memory, Array{TableReader.Token, 2}, Int64, UInt8})
-    precompile(Tuple{typeof(TableReader.parse_datetime), Array{String, 1}})
-    precompile(Tuple{typeof(TableReader.allocate!), TableReader.StringCache, Ptr{UInt8}, Int64})
-    precompile(Tuple{typeof(TableReader.skiplines), TranscodingStreams.TranscodingStream{TranscodingStreams.Noop, Base.IOStream}, Int64})
+    precompile(Tuple{typeof(TableReader.checkformat), TranscodingStreams.TranscodingStream{TranscodingStreams.Noop, Base.Process}})
+    precompile(Tuple{typeof(TableReader.parse_datetime), Array{String, 1}, Bool})
     precompile(Tuple{typeof(TableReader.wrapstream), Base.Process, TableReader.ParserParameters})
     precompile(Tuple{typeof(TableReader.wrapstream), Base.IOStream, TableReader.ParserParameters})
-    precompile(Tuple{typeof(TableReader.fillcolumn!), Array{String, 1}, Int64, TranscodingStreams.Memory, Array{TableReader.Token, 2}, Int64, UInt8})
-    precompile(Tuple{typeof(TableReader.is_date_like), Array{String, 1}})
+    precompile(Tuple{typeof(TableReader.checkformat), Base.Process})
+    precompile(Tuple{typeof(TableReader.allocate!), TableReader.StringCache, Ptr{UInt8}, Int64})
     precompile(Tuple{typeof(TableReader.is_datetime_like), Array{String, 1}})
+    precompile(Tuple{typeof(TableReader.is_date_like), Array{String, 1}})
+    precompile(Tuple{typeof(TableReader.fillcolumn!), Array{String, 1}, Int64, TranscodingStreams.Memory, Array{TableReader.Token, 2}, Int64, UInt8})
+    precompile(Tuple{typeof(TableReader.parse_datetime), Array{Union{Base.Missing, String}, 1}, Bool})
+    precompile(Tuple{typeof(TableReader.countbytes), TranscodingStreams.Memory, UInt8})
+    precompile(Tuple{typeof(TableReader.parse_date), Array{Union{Base.Missing, String}, 1}})
+    precompile(Tuple{getfield(TableReader, Symbol("##readcsv#14")), Char, Char, Bool, Bool, Int64, Bool, Nothing, Bool, Bool, Int64, typeof(TableReader.readcsv), String})
+    precompile(Tuple{typeof(TableReader.normalizename), String})
+    precompile(Tuple{typeof(TableReader.skiplines), TranscodingStreams.TranscodingStream{TranscodingStreams.Noop, Base.IOStream}, Int64})
+    precompile(Tuple{getfield(TableReader, Symbol("##readtsv#23")), Char, Char, Bool, Bool, Int64, Bool, Nothing, Bool, Bool, Int64, typeof(TableReader.readtsv), String})
     precompile(Tuple{typeof(TableReader.readcsv), String})
     precompile(Tuple{typeof(TableReader.readtsv), String})
-    precompile(Tuple{getfield(TableReader, Symbol("##readcsv#12")), Char, Char, Bool, Int64, Bool, Nothing, Int64, typeof(identity), String})
-    precompile(Tuple{getfield(TableReader, Symbol("##readtsv#21")), Char, Char, Bool, Int64, Bool, Nothing, Int64, typeof(identity), String})
 end
-_precompile_()
```

Comment:
Please do not remove this line. This precompiles methods.

Reply:
fixed in latest commit